src/components/generate/generate.ts

```diff
@@ -0,0 +1,290 @@
+import { API, CarTrack, Control, Race, App, Winners } from '../index';
+
+export class Generate {
```

Comment:
bad naming from my sight
generate what
it would also be better to name it at least with a noun
you may read about the naming conventions